src/index.js

```diff
@@ -0,0 +1,55 @@
+import express from 'express';
+import cors from 'cors';
+import { json } from 'body-parser';
+import log from './utils/log';
+import db from './db';
+import v1Router from './routes/v1';
+import HttpError from './exceptions/HttpError';
+
+require('dotenv').config();
```

Comment:
I wonder if we could do:

import * as dotEnv from 'dotenv';
dotEnv.config()

Another one on not mixing ESM and require()